src/__tests__/firstLetters.test.js

```diff
@@ -51,6 +51,11 @@ describe('firstLetters', () => {
       .toBe('ddmm');
   });
 
+  it('Should return first letters of each word, including ਙ (|)', () => {
+    expect(firstLetters('|M|w i|Awnu nhI muK bwqau ]'))
+      .toBe('||nmb');
+  });
+
```

Comment:
Test for the original reason that the pipe char replacement was added:
```suggestion
  it('Should return first letters of each word, including ਙ (|)', () => {
    expect(firstLetters('|M|w i|Awnu nhI muK bwqau ]'))
      .toBe('||nmb');
  });
  
  it('Should return first letters of each word, excluding the pipe char (|) when input is English', () => {
    expect(firstLetters('|| Jap ||', true))
      .toBe('j');
  });
```